app/controllers/api/v1/rental_assistances_controller.rb

```diff
@@ -24,7 +24,8 @@ def update
         )
 
         if response
-          render json: true
+          updated_rental_assistances = soql_rental_assistance_service.application_rental_assistances(params[:application_id])
```

Comment:
one small nit is that I've been updating other similar methods to use the bang (!) methods to actually raise an error that we can see in the logs instead of having this 422 status response. That way if something goes wrong, the user experience isn't actually that different, and I can see the logs in papertrail